src/find-specs.js

```diff
@@ -254,26 +256,151 @@ const hasPublishedContent = (candidate) => fetch(candidate.spec).then(({ok, url}
     }
   }
   candidates = candidates.filter(candidate => !!candidate.spec);
+  for (const candidate of candidates) {
+    try {
+      candidate.shortname = computeShortname(candidate.spec).shortname;
+    }
+    catch {}
+  }
+  candidates.sort((c1, c2) => {
+    if (c1.shortname && c2.shortname) {
+      return c1.shortname.localeCompare(c2.shortname);
+    }
+    else if (c1.shortname) {
+      return -1;
+    }
+    else if (c2.shortname) {
+      return 1;
+    }
+    else {
+      return c1.spec.localeCompare(c2.spec);
+    }
+  });
+
+  return {
+    additions: candidates,
+    monitor: monitorAdditions
+  };
+}
+
 
-  const candidate_list = candidates.sort((c1, c2) => c1.spec.localeCompare(c2.spec))
-        .map(c => `- [ ] ${c.spec} from [${c.repo}](https://github.com/${c.repo})` + (c.impl.chrome ? ` [chrome status](https://www.chromestatus.com/features/${c.impl.chrome})` : '')).join("\n");
-  core.exportVariable("candidate_list", candidate_list);
-  console.log();
-  console.log(candidate_list);
-  if (monitorAdditions.length) {
-    const today = new Date().toJSON().slice(0, 10);
-    const monitored = monitorAdditions.map(({repo}) => `- [ ] [${repo}](https://github.com/${repo})`).join("\n");
-    core.exportVariable("monitor_list", monitored);
-    monitorAdditions.forEach(({repo}) => {
-      monitorList.repos[repo] = {
-        lastreviewed: today,
-        comment: "no published content yet"
-      };
-    });
-    fs.writeFileSync("./src/data/monitor.json", JSON.stringify(monitorList, null, 2));
-    console.log(monitored);
+function toInt(value) {
```

Comment:
since this is very specific to being used in the context of the CLI validation, maybe the function name could be more specific too